10_install_server/11_install_root_server_debian.txt

```diff
@@ -38,16 +38,13 @@ apt-get install rudder-server-root
 
 ----
 
-Now jump to <<initial-config,the next section to configure your server>>.
-
 [WARNING]
 
 ====
 
-For users running the Rudder *server* on Ubuntu Server 12.04 or later, any
-nodes running syslogd (not syslog-ng or rsyslog) will fail to send any
-reports about the configuration rules they have applied. This is the case by
-default on RHEL/CentOS 5, but not on any other supported platforms.
+Any nodes running *syslogd* (not syslog-ng or rsyslog) will *fail* to send any
+reports about the configuration rules they have applied to a Rudder Server running
+on Ubuntu.
```

Comment:
I agree that your re-formulation is better, but please emphasize more that this only applies to Rudder servers on **Ubuntu** and that if your server is on **any other operating system** this is not a problem.